src/map/map.hpp

```diff
@@ -344,6 +344,19 @@ enum e_element : int8{
 	ELE_RANDOM,
 };
 
+static std::unordered_map<std::string, e_element> um_eleid2elename {
```

Comment:
Why "static"?

Reply:
hi @Lemongrass3110 , im using debian, im getting error with this line, 

![Screenshot 2021-08-21 221823](https://user-images.githubusercontent.com/31005928/130353627-b2f94fb4-4694-4a69-8070-ce6af943c926.png)

i think it has something to do with the string not being defined.